released/packages/coq-library-undecidability/coq-library-undecidability.1.1.2+8.20/opam

```diff
@@ -0,0 +1,37 @@
+opam-version: "2.0"
+maintainer: "forster@ps.uni-saarland.de"
+homepage: "https://github.com/uds-psl/coq-library-undecidability/"
+dev-repo: "git+https://github.com/uds-psl/coq-library-undecidability/"
+bug-reports: "https://github.com/uds-psl/coq-library-undecidability/issues"
+authors: ["Yannick Forster"
+          "Dominique Larchey-Wendling"
+          "Andrej Dudenhefner"
+          "Edith Heiter"
+          "Marc Hermes"
+          "Johannes Hostert"
+          "Dominik Kirst"
+          "Mark Koch"
+          "Fabian Kunze"
+          "Gert Smolka"
+          "Simon Spies"
+          "Dominik Wehr"
+          "Maxi Wuttke"]
+
+license: "MPL-2.0"
+build: [
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+depends: [
+  "coq" {>= "8.20" & < "8.21~"}
+  "ocaml"
```

Comment:
```suggestion
```
@yforster from the repo, it looks like this is a pure Coq project (thanks to MetaCoq). So the OCaml dependency is probably not needed?